Bio/SubsMat/__init__.py

```diff
@@ -601,7 +618,7 @@ def two_mat_relative_entropy(mat_1, mat_2, logbase=2, diag=diagALL):
         if mat_1[key] > EPSILON and mat_2[key] > EPSILON:
             val_1 = mat_1[key] / sum_ent_1
             val_2 = mat_2[key] / sum_ent_2
-#            rel_ent += mat_1[key] * log(mat_1[key]/mat_2[key])/log(logbase)
+            #            rel_ent += mat_1[key] * log(mat_1[key]/mat_2[key])/log(logbase)
```

Comment:
The indentation of this comment looks strange now - do you think we can just remove it?

Reply:
I have removed it, is been commented for more than 10 Years, I think it didn't calculate the relative entropy properly so got commented.